crutest/src/main.rs

```diff
@@ -84,6 +84,8 @@ enum Workload {
         skip_verify: bool,
     },
     Generic,
+    /// Do random read IOs
```

Comment:
```suggestion
    /// Do random read and flush IOs
```
(to distinguish between `RandRead` below)

Reply:
Done